versions.txt

```diff
@@ -13,7 +13,6 @@ google-cloud-bigquery:1.62.0:1.62.1-SNAPSHOT
 google-cloud-bigquerydatatransfer:0.80.0-beta:0.80.1-beta-SNAPSHOT
 google-cloud-bigquerystorage:0.80.0-beta:0.80.1-beta-SNAPSHOT
 google-cloud-bigtable:0.80.0-alpha:0.80.1-alpha-SNAPSHOT
-google-cloud-bigtable-admin:0.80.0-alpha:0.80.1-alpha-SNAPSHOT
```

Comment:
what about proto-google-cloud-bigtable-admin-v2:0.45.0:0.45.1-SNAPSHOT and grpc?

Reply:
Those should stay.  Only the gapic client was merged

Reply:
The generated protos & grpc stubs, are still published separately:
https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-bigtable/synth.py#L46-L47

While the gapic library is merged:
https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-bigtable/synth.py#L45